python/setup.py

```diff
@@ -62,8 +66,71 @@ def get_build_version():
     return version
 
 
-LIB_LIST = get_lib_path()
-__version__ = get_build_version()
+# FIXME: commented out LIB_LIST because it relies on get_lib_path()
+# LIB_LIST = get_lib_path()
+
+# FIXME: commented out _version_ because subprocess.CalledProcessError: Command '['git', 'rev-parse', '--short', 'HEAD']' returned non-zero exit status 128.
+# __version__ = get_build_version()
+
+__version__ = "0.1"
+
+
+def config_cython():
+    """Try to configure cython and return cython configuration"""
+    if FFI_MODE not in ("cython"):
+        if os.name == "nt" and not CONDA_BUILD:
+            print("WARNING: Cython is not supported on Windows, will compile without cython module")
+            return []
+        sys_cflags = sysconfig.get_config_var("CFLAGS")
+        if sys_cflags and "i386" in sys_cflags and "x86_64" in sys_cflags:
+            print("WARNING: Cython library may not be compiled correctly with both i386 and x64")
+            return []
+    try:
+        from Cython.Build import cythonize
+
+        # from setuptools.extension import Extension
```

Comment:
remove or uncomment?

Reply:
Thanks, I will remove it as we import Extension at the beginning of the file.